src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: fit-content;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-size: cover;
+}
+
+.stars--0 .stars__star:nth-child(-n+0),
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
```

Comment:
The `nth-child` selectors are using a pattern `-n+X` where X is the number of stars to be styled. However, the pattern `-n+0` in line 17 will not select any elements, which might not be the intended behavior. You might want to adjust this to ensure it selects the correct number of stars.